### PR TITLE
Jenkins: Java 8 JDK installation

### LINKED ISSUE #180: jenkins: install java 8 jdk
We should preinstall the java 8 jdk as jenkins tool (like we did with maven).

### PR DESCRIPTION
**Pull request by robertauer**
resolves #180

### CODE REVIEW

File: containers/jenkins/resources/init.groovy.d/08-java-autoinstall.groovy [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import hudson.model.JDK
+import hudson.tools.JDKInstaller
+import hudson.tools.InstallSourceProperty
+import jenkins.model.Jenkins
+
+def descriptor = new JDK.DescriptorImpl();
+
+if (!descriptor.getInstallations()) {
+    Jenkins.instance.updateCenter.getById('default').updateDirectlyNow(true)
```
Comment (sdorra) [inline containers/jenkins/resources/init.groovy.d/08-java-autoinstall.groovy]: Do we really need a update of the update center? Why is this required?

**Comment by robertauer** [comment]
Update of the updatecenter was not required.
